tests/TestHelpers.ts

```diff
@@ -0,0 +1,32 @@
+import {GraphQLRequestInfo,
+    GraphQLServerOptions,
+    JsonLogger} from '../src';
+import {userSchema,
+    userSchemaResolvers} from './ExampleSchemas';
+import fetch from 'cross-fetch';
+
+export const GRAPHQL_SERVER_PORT = 3000
+export const LOGGER = new JsonLogger('test-logger', 'myTestService')
+export const INITIAL_GRAPHQL_SERVER_OPTIONS: GraphQLServerOptions = {schema: userSchema, rootValue: userSchemaResolvers, logger: LOGGER, debug: true}
+
+export function generateGetParamsFromGraphQLRequestInfo(requestInfo: GraphQLRequestInfo): string {
+    let result=''
```

Comment:
Formatting: I think there should be spaces around the assignment operator, or shouldn't it?

Reply:
Fixed